api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -787,10 +875,10 @@ type MachineNetworkEntry struct {
 // ClusterNetworkEntry is a single IP address block for pod IP blocks. IP blocks
 // are allocated with size 2^HostSubnetLength.
 type ClusterNetworkEntry struct {
-	// CIDR is the IP block address pool.
+	// cidr is the IP block address pool.
 	CIDR ipnet.IPNet `json:"cidr"`
```

Comment:
What schema is generated for this type? Can you validate the fact that this is a real CIDR?
```
isCIDR(self)
```